packages/utils/src/debugTimer.ts

```diff
@@ -1,7 +1,7 @@
 const debugNamesOngoing = new Set<string>();
 
 export function debugTimerStart(name: string) {
-  const debugEnvVar = globalThis?.process.env['DEBUG'] || (globalThis as any).DEBUG;
+  const debugEnvVar = globalThis.process.env['DEBUG'] || (globalThis as any).DEBUG;
```

Comment:
Yes I agree, but the error is not due to my change. It also exists in the current version.

![image](https://github.com/user-attachments/assets/ae5cbe5c-148c-4621-b0c4-6228d983bc4d)

Reply:
> It seems a bug in the bundler you shared.

Yes definitely, Cloudflare workers cannot handle the `globalThis?.` expression.

Reply:
I think I misexplained myself. `process` is not a standard in JavaScript so `process` might not be available in `globalThis`.
If we merge and release your change, the environments that don't have `process` in `globalThis` will fail because without optional chaining, the code will always expect `process` to be there :) I hope it is clear now.

Reply:
Sorry if I understand something wrong but in my opinion the current code also fails.

`undefined is not an object (evaluating 'globalThis?.process.env')`

When `process` is not available in `globalThis` it tries to execute `globalThis.undefined.env`.

The image I posted above demonstrates this. It would only not fail if there were *two* optional chaining.
`globalThis?.process?.env['DEBUG']`

Reply:
Oh now I see. The problem is `globalThis?.` Ok ok :) My bad.